tests/h/auth/worker_test.py

```diff
@@ -34,8 +34,28 @@ def test_it_leaves_valid_tickets(self, db_session, factories):
         worker.delete_expired_auth_tickets()
         assert db_session.query(AuthTicket).count() == 1
 
-    @pytest.fixture
-    def celery(self, patch, db_session):
-        cel = patch('h.auth.worker.celery', autospec=False)
-        cel.request.db = db_session
-        return cel
+
+@pytest.mark.usefixtures('celery')
+class TestDeleteExpiredTokens(object):
+    def test_it_removes_expired_tokens(self, db_session, factories):
+        factories.Token(expires=datetime(2014, 5, 6, 7, 8, 9))
+        factories.Token(expires=(datetime.utcnow() - timedelta(seconds=1)))
+
+        assert db_session.query(Token).count() == 2
+        worker.delete_expired_tokens()
+        assert db_session.query(Token).count() == 0
+
+    def test_it_leaves_valid_tickets(self, db_session, factories):
+        factories.Token(expires=datetime(2014, 5, 6, 7, 8, 9))
+        factories.Token(expires=(datetime.utcnow() + timedelta(hours=1)))
+
+        assert db_session.query(Token).count() == 2
+        worker.delete_expired_tokens()
+        assert db_session.query(Token).count() == 1
```

Comment:
Could we add a test that a token with `expires=None` isn't deleted, because those are developer tokens and it would be a bit embarrassing if we blew them all away.